src/guess-number.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const readline = require('readline'),
+rl = readline.createInterface(process.stdin, process.stdout)
+
+export const GuessNum = {
+   generateRandomNumber() {
+    let randomNum = Math.floor(Math.random()*100)
```

Comment:
Indentation should be consistent in your files.  Best practice in javascript is a consistent, two space indentation.